app/models/lightweight_activity.rb

```diff
@@ -210,6 +210,7 @@ def self.extract_from_hash(activity_json_object)
   end
 
   def self.import(activity_json_object,new_owner,imported_activity_url=nil,helper=nil)
+    version = activity_json_object[:version] || 1
```

Comment:
Seems like you need to do this on the sequence model too? Or is the version specified on each of the individual activities (I faintly remember there being something weird like that)

Reply:
Yeah, unfortunately it is not set at the top level of the sequence but rather in each activity.